kubernetes/linux/main.sh

```diff
@@ -169,6 +169,15 @@ cat health_config_env_var | while read line; do
 done
 source health_config_env_var
 
+#Parse the configmap to set the right environment variables for network policy manager (npm) integration.
+/opt/microsoft/omsagent/ruby/bin/ruby tomlparser-npm-config.rb
+
+cat integration_npm_config_env_var | while read line; do
+    #echo $line
+    echo $line >> ~/.bashrc
+done
+source integration_npm_config_env_var
```

Comment:
curios, why are refreshing the integration_npm_config_env_var as oppose to bashrc here since the envs are written to bashrc and being used in the telegraf conf files?

Reply:
its because we are sourcing bashrc at the end, when these vars are needed in the current session right away, its sourced before bashrc.